pom.xml

```diff
@@ -772,6 +772,32 @@
       <properties>
         <skipTests>${release.skipTests}</skipTests>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-source-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>attach-sources</id>
+                <goals>
+                  <goal>jar-no-fork</goal>
```

Comment:
Yet you are not using `javadoc-no-fork`—intentional?

The [superpom](https://maven.apache.org/ref/3.3.3/maven-model-builder/super-pom.html) seems to use `jar` for both. I do not understand the fine points here, just asking if there is a reason you are diverging from the superpom.